lightning_pose/train.py

```diff
@@ -168,7 +176,9 @@ def train(cfg: DictConfig) -> None:
     # Post-training analysis
     # ----------------------------------------------------------------------------------
     # get best ckpt
-    best_ckpt = os.path.abspath(trainer.checkpoint_callback.best_model_path)
+    best_ckpt = ckpt_path_from_base_path(
```

Comment:
This change allowed me to comment out trainer.fit to quickly regenerate detector artifacts.